english/bird-salvation/bird.md

```diff
@@ -10,3 +10,7 @@ what appears to be a horse with a horn sprouting from its forehead...
 You really have only one option
 [Go ask Alice](../ask-to-alice/ask-to-alice.md)
 I think she'll know ...
+
+The pressing question is whether Alice can help in time to avoid nightfall and the inevitable zombies that sprout from dark corners. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.